asciidoc-confluence-publisher-cli/src/it/java/org/sahli/asciidoc/confluence/publisher/cli/AsciidocConfluencePublisherCommandLineClientIntegrationTest.java

```diff
@@ -21,36 +21,61 @@
 
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.is;
 
 public class AsciidocConfluencePublisherCommandLineClientIntegrationTest {
 
+    private static final String CONFLUENCE_URL = "localhost:8090";
```

Comment:
I am not convinced that extracting those values into constants is increasing readability. It breaks the reading flow as each time, I would need to get back to understand which value ist passed. We usually prefer working with literals in tests.

Reply:
In IntelliJ Idea you need to hover the cursor over the constant in code and press Cmd and you will get popup with declaration of that value:
<img width="711" alt="Screen Shot 2019-03-20 at 6 17 54 PM" src="https://user-images.githubusercontent.com/1207414/54700887-869bf700-4b3c-11e9-9ce0-229ddf0a5f98.png">
When you need to change the property value -- you need to go all over the code and touch multiple places instead of changing one constant.
I will revert the changes, it is not an issue for me.

Reply:
Thanks for the hint ;-)